modules/ROOT/pages/values-and-types/property-structural-constructed.adoc

```diff
@@ -104,6 +105,56 @@ However, not all types can be used in all places.
 All Cypher types contain the `null` value. To make them not nullable, `NOT NULL` can be appended to the end of the type (e.g. `BOOLEAN NOT NULL`, `LIST<FLOAT NOT NULL>`).
 Note that closed dynamic types (`INNER_TYPE_1 | INNER_TYPE_2...`) cannot be appended with `NOT NULL`: all inner types must be nullable, or all appended with `NOT NULL`.
 
+== Type Normalization
+
+Cypher runs a normalization algorithm on all input types, simplifying the given type to a deterministic representation for equivalent types.
+Types are simplified to their default name (e.g. `BOOL` is simplified to `BOOLEAN`), encompassing types are absorbed (e.g. `LIST<BOOLEAN> | LIST<BOOLEAN | INTEGER>` is normalized to `LIST<BOOLEAN | INTEGER>`) and they are <<ordering-of-types, ordered>>.
```

Comment:
xrefs are easier to spot in case we move material and the link breaks.
Also, I think breaking this up into smaller sentences makes it easier to read. 
Do we need to clarify what "encompassing types" are? I think I understand, but not 100% to be honest. 

```suggestion
Types are simplified to their default name (e.g. `BOOL` is simplified to `BOOLEAN`). 
Encompassing types are absorbed (e.g. `LIST<BOOLEAN> | LIST<BOOLEAN | INTEGER>` and normalized to `LIST<BOOLEAN | INTEGER>`).
Types are also xref:values-and-types/property-structural-constructed.adoc#ordering-of-types[ordered].
```

Reply:
An encompassing type is one that contains all the types of another type, I am happy to add more here but not sure exactly what I'd write :)

Reply:
I think it's probably fine as is - that was how I understood it without any further descriptions.